documentation/client/docs/libraries/python/references/iota_client/client.md

```diff
@@ -82,7 +82,18 @@ Generate addresses.
 ### build\_and\_post\_block
 
 ```python
-def build_and_post_block(secret_manager=None, options=None)
+def build_and_post_block(secret_manager=None, 
```

Comment:
The actual `build_and_post_block` function is not changed in this PR ?

Reply:
God damnit. That probably has been lost in one of the latest rebases. Sorry.
Let me check 😓

Reply:
wen